proxy/http/remap/RemapHitCount.cc

```diff
@@ -0,0 +1,90 @@
+/** @file
+
+  A brief file description
```

Comment:
Perhaps you could be more explicit here.

Reply:
Ack, fixed.